src/core/api/HTTPTransport.ts

```diff
@@ -0,0 +1,92 @@
+import { AnyType } from '../../types';
+
+export enum Method {
+  Get = 'Get',
+  Post = 'Post',
+  Put = 'Put',
+  Patch = 'Patch',
+  Delete = 'Delete',
+}
```

Comment:
```suggestion
export type Method =
| 'GET'
| 'POST'
| 'PUT'
| 'DELETE'
| 'PATCH'   
```

union в TS - очень мощный инструмент

Reply:
поменял на тип